mypy/main.py

```diff
@@ -866,6 +872,26 @@ def process_cache_map(parser: argparse.ArgumentParser,
         options.cache_map[source] = (meta_file, data_file)
 
 
+def split_and_match_files(paths: str) -> List[str]:
+    """Take a string representing a list of files/directories (including globbing with */**).
+
+    Where a path/glob matches no file, we still include the raw path in the resulting list.
+
+    Returns a list of file paths
+    """
+    expanded_paths = []
+
+    for path in re.split('[,:]', paths):
```

Comment:
Is the colon here due it being the path separator on Unix? If so, it might be good to also add semicolon, as that is the path separator on Windows.

Reply:
I copied this re.split from the lambda on mypy_path a few lines down. Happy to add the semicolon as well; would it make sense to add it there too?

Reply:
Honestly I wonder if it should just be comma, or space and comma? This isn't a search path like MYPYPATH, and when I tried to test this the very first mistake I made was using a space for a separator.

(I should add that for mypy_path, the behavior for `$MYPYPATH` is different than that for the config `mypy_path` variable -- the latter always uses `[,:]`, while the former uses `os.pathsep`, which is only `:` on *NIX and only `;` on Windows. So I don't think that's a great model.)

Reply:
I'm a smidge nervous about using spaces as separators as that seems the most likely of all to be legitimately used in a path (albeit hopefully unlikely), and managing those within paths would introduce a lot more complexity. I think probably just commas are sufficient? It will always be processed slightly differently than by a shell, so making a clear distinction feels reasonable.

Reply:
I guess that might be the case on Windows... Agreed though, so let's do just comma. (I suppose colon is also sketchy on Windows, as in `C:\`.)